src/routes/product.route.js

```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../db/db.client");
+const Product = db.product;
+
+// Public
+// GET all
+router.get("/", async (req, res) => {
+  try {
+    const products = await Product.findAll();
+    res.json(products);
+  } catch (err) {
+    console.error(err.message);
+    res.status(500).send("Server Error");
```

Comment:
move to constant file http_codes 
constants/httpCodes.js
export const SERVER_ERROR = 500;

const {SERVER_ERROR} = require('const/httpcodes');

Reply:
* as httpcodes;

Reply:
mvoe to error messages

Reply:
done